posts/specfile-introduction/index.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Introduction to specfile library
+date: 2024-01-12
+authors: nforro
+tags:
+  - specfile
+---
+
+# Introduction to specfile library
+
+Have you ever wanted to make changes in a RPM spec file programatically? _specfile_ library
+has been created for that very purpose. It is a pure Python library that allows you to conveniently
+edit different parts of a spec file while doing its best to keep the resulting changeset minimal
+(no unnecessary whitespace changes etc.).
+
+## Installation
+
+The library is packaged for Fedora, EPEL 9 and EPEL 8 and you can simply instal it with dnf:
+
+```bash
+dnf install python3-specfile
+```
+
+On other systems, you can use pip (just note that it requires RPM Python bindings to be installed):
```

Comment:
```suggestion
On the other systems, you can use pip (just note that it requires RPM Python bindings to be installed):
```

Reply:
Are you sure about this one? I just doesn't sound right to me with the article :slightly_smiling_face: